client/app/l10n/en.json

```diff
@@ -421,7 +421,7 @@
   "No submission was found for the specified ID": "No submission was found for the specified ID",
   "No Submission was found for the specified ID": "No Submission was found for the specified ID",
   "None": "None",
-  "Not accepted": "Not accepted",
+  "Not accepted": "Declined",
```

Comment:
This texts is in relation to a checkbox that could be Accepted or Not accepted. We considered keep the current text for the moment until eventually we will change what communicated entirely.